app/code/Magento/Sales/Block/Adminhtml/Order/View.php

```diff
@@ -315,7 +314,11 @@ public function getHeaderText()
      */
     public function getUrl($params = '', $params2 = [])
     {
-        $params2['order_id'] = $this->getOrderId();
+        if (isset($params2['action']) && $params2['action'] === 'index') {
```

Comment:
Do we still need this part? We don't use `getUrl()` method for the back button since the last change.

Reply:
Sorry, missed it. Have made the changes and updated the code.